db/sql.h

```diff
@@ -1251,8 +1217,6 @@ int get_prepared_stmt(struct sqlthdstate *, struct sqlclntstate *, struct sql_st
 int get_prepared_stmt_no_lock(struct sqlthdstate *, struct sqlclntstate *, struct sql_state *, struct errstat *, int);
 int get_prepared_stmt_try_lock(struct sqlthdstate *, struct sqlclntstate *, struct sql_state *, struct errstat *, int);
 
-void put_prepared_stmt(struct sqlthdstate *, struct sqlclntstate *,
-                       struct sql_state *, int outrc);
 void sqlengine_thd_start(struct thdpool *, struct sqlthdstate *, enum thrtype);
 void sqlengine_thd_end(struct thdpool *, struct sqlthdstate *);
 void delete_stmt_caching_table(hash_t *stmt_caching_table);
```

Comment:
does delete_stmt_caching_table() still exist?

Reply:
No, it shouldn't, removed.